domain/application/state/storage.go

```diff
@@ -73,3 +77,55 @@ VALUES ($storageToAdd.*)`, storageToAdd{})
 	}
 	return nil
 }
+
+// GetStorageUUIDByID returns the UUID for the specified storage, returning an error
```

Comment:
In an ideal world  this would be introduced in a seperate PR to keep this ones size down and concerns isolated.